bigbang/mailman.py

```diff
@@ -122,8 +66,8 @@ def collect_from_url(
     if has_archives:
         unzip_archive(url)
         try:
-            data = open_list_archives(url)
-        except MissingDataException as e:  # don't strictly need to open the archives during the collection process, so catch the Exception and return
+            data = archive.open_list_archives(url)
+        except archive.MissingDataException as e:  # don't strictly need to open the archives during the collection process, so catch the Exception and return
             print(e)
```

Comment:
I think this should be `logging.warning` or `logging.exception` or something, rather than relying on printing to stdout.